packages/core/src/components/TextField.tsx

```diff
@@ -427,27 +436,22 @@ class TextField extends React.Component<Props, State> {
           <Icon {...leftIconProps} style={leftIconStyle} />
         ) : null}
         <View
-          style={[
-            containerStyle,
-            style
-              ? {
-                  ...(style.height ? { height: style.height } : {}),
-                  ...(bgColor ? { backgroundColor: bgColor } : {}),
-                  ...(padding ? { padding } : {}),
-                  ...(paddingTop ? { paddingTop } : {}),
-                  ...(paddingBottom ? { paddingBottom } : {}),
-                  ...(paddingLeft ? { paddingLeft } : {}),
-                  ...(paddingRight ? { paddingRight } : {}),
-                  ...(borderRadius ? { borderRadius } : {}),
-                  ...(borderWidth ? { borderWidth } : {}),
-                  ...(borderTopWidth ? { borderTopWidth } : {}),
-                  ...(borderRightWidth ? { borderRightWidth } : {}),
-                  ...(borderBottomWidth ? { borderBottomWidth } : {}),
-                  ...(borderLeftWidth ? { borderLeftWidth } : {}),
-                  ...(borderCol ? { borderColor: borderCol } : {}),
-                }
-              : {},
-          ]}
+          style={applyStyles(StyleSheet.flatten(containerStyle), {
```

Comment:
do you think we can include the flatten inside applyStyles?

Reply:
Definitely - made this change